src/modules/dashboard/components/Extensions/Extensions.css

```diff
@@ -42,3 +43,52 @@
 .loadingSpinner {
   margin-top: 100px;
 }
+
+@media screen and query700 {
+  .main {
+    display: flex;
+    flex-direction: column;
+    padding-bottom: 60px;
+  }
+
+  .description {
+    margin: 0;
+    padding: 0 14px;
+  }
+
+  .main hr {
+    height: 0.8px; /* 1px looks thicker than the borders on the other elements. */
+    background-color: var(--temp-grey-13);
+  }
+
+  .main span[class*="breadCrumble"] {
+    margin-bottom: 20px;
+    font-family: var(--family-primary);
+    font-size: var(--size-medium-l);
+    font-weight: var(--weight-bold);
+    line-height: 1.5;
+    word-break: normal;
+    white-space: pre-line;
+    color: var(--dark);
+  }
```

Comment:
I've copied these styles from `Heading.themeDark`. Can't use `compose` because the selector is complex.